config/monitoring/metrics/prometheus/100-prometheus-scrape-config.yaml

```diff
@@ -379,3 +379,39 @@ data:
         target_label: pod
       - source_labels: [__meta_kubernetes_service_name]
         target_label: service
+
+    - job_name: kubernetes-service-endpoints
+      kubernetes_sd_configs:
+      - role: endpoints
+      relabel_configs:
+      - action: keep
+        regex: true
+        source_labels:
+        - __meta_kubernetes_service_annotation_prometheus_io_scrape
+      - action: replace
+        regex: (https?)
+        source_labels:
+        - __meta_kubernetes_service_annotation_prometheus_io_scheme
```

Comment:
You didn't mention annotation `prometheus.io/path` in #4616 but it is fetched as a label here. Do you intend to use it? If no, I'd prefer not to add it in this PR.

Reply:
Sure. delete it. Thanks.